javascript-practice/src/js/constants/config.js

```diff
@@ -0,0 +1,6 @@
+const API_ROUTES = {
```

Comment:
When naming a file, describe its purpose

This file is a content API route, so you can use names such as`url-api.js`, `api-endpoint.js` ......
the name `config.js` is not clear